docs/pages/table.js

```diff
@@ -145,90 +75,1666 @@ card(
+          <MainSection.Card
+            cardSize="md"
+            type="do"
+            description={`Align content so that it’s easy to scan, read and compare:
+- Start-align text and combo-content (combinations of text, numbers and/or graphics)
+- End-align numbers only
+- Align headers with their corresponding content
+- Use tabular lining for numbers`}
+            defaultCode={`
+function Example() {
+  const HeaderRow = ({ id }) => {
+    return (
+      <Table.Header>
+        <Table.Row>
+          <Table.HeaderCell>
+            <Box display="visuallyHidden"><Label htmlFor={id}>Not all checkboxes are checked</Label></Box>
+            <Checkbox
+              id={id}
+   onChange={() => {}}
+              indeterminate
+              size="sm"
+            />
+          </Table.HeaderCell>
+          {["Campaign", "Spend"].map((title, idx) => {
+            return (
+              <Table.HeaderCell key={idx}>
+                <Text align={title === "Spend" ? "end" : "start"} weight="bold">{title}</Text>
+              </Table.HeaderCell>
+            );
+          })}
+        </Table.Row>
+      </Table.Header>
+    );
+  };
+
+  const BaseRow = ({ id, checked, disabled, text, spend }) => {
+    return (
+      <Table.Row>
+        <Table.Cell>
+          <Checkbox
+            id={id.replace(/ /g, "_").replace(/'/g, "") + "_" + text.replace(/ /g, "_").replace(/'/g, "")} onChange={() => {}}
+            disabled={disabled}
+            size="sm"
+            checked={checked}
+          />
+        </Table.Cell>
+        <Table.Cell>
+          <Label htmlFor={id.replace(/ /g, "_").replace(/'/g, "") + "_" + text.replace(/ /g, "_").replace(/'/g, "")}>
+            <Text>{text}</Text>
+          </Label>
+        </Table.Cell>
+        <Table.Cell>
+          <Text align="end">{spend}</Text>
+        </Table.Cell>
+      </Table.Row>
+    );
+  };
+
+  const tableID = "Example of a 'do' for table alignment";
+
+  return (
+    <Table accessibilityLabel={tableID}>
+      <HeaderRow id={tableID}/>
+      <Table.Body>
+       <BaseRow
+          id={tableID}
+          checked={true}
+          text="The best ad"
+          spend="$5.50"
+        />
+        <BaseRow
+          id={tableID}
+          disabled
+          text="This ad is great"
+          spend="$3,000.00"
+        />
+        <BaseRow
+          id={tableID}
+          checked={true}
+          text="Macy's pincycle"
+          spend="$1.75"
+        />
+        <BaseRow
+          id={tableID}
+          checked={false}
+          text="Best Buy Wins"
+          spend="$51,650,500.54"
+        />
+        <BaseRow
+          id={tableID}
+          checked={false}
+          text="The third campaign"
+          spend="$67.60"
+        />
+      </Table.Body>
+    </Table>
+  );
+}
+`}
+          />
+          <MainSection.Card
+            cardSize="md"
+            type="don't"
+            description={`Don’t
```

Comment:
Need to remove the "Don't" in description

Reply:
DONE